Unable to save settings (Iframe integration) 0.49

Your Rocket.Chat version: (0.49)

I'm trying to activate iframe integration in (general->iframe integration).
While settings both option to true (receive and send events to parent windows) I see the save settings button at the top right but nothing happens when clicking it. no error message is displayed in the server or client console. nothing is saved.

Other settings such as setting the site name work or other accounts settings.

However I'm getting this kind of messages in the server console : 
`users Falling back _publishCursor to model with query: { username: 'admin' }`
is it linked in any way ? 

and this is what I get right after starting rocket chat : 

```
I20161226-12:43:12.601(1)? Will load cache for users
I20161226-12:43:12.669(1)? 2 records load from users
I20161226-12:43:12.669(1)? Will load cache for rocketchat_room
I20161226-12:43:12.670(1)? 1 records load from rocketchat_room
I20161226-12:43:12.670(1)? Will load cache for rocketchat_subscription
I20161226-12:43:12.671(1)? 1 records load from rocketchat_subscription
I20161226-12:43:12.671(1)? Will load cache for rocketchat_settings
I20161226-12:43:12.671(1)? 426 records load from rocketchat_settings
I20161226-12:43:12.672(1)? rocketchat_settings Falling back observe to model with query: undefined
I20161226-12:43:12.672(1)? rocketchat_settings Falling back observe to model with query: undefined
I20161226-12:43:12.672(1)? Updating process.env.MAIL_URL
I20161226-12:43:12.672(1)? Will load cache for rocketchat_permissions
I20161226-12:43:12.673(1)? 63 records load from rocketchat_permissions
I20161226-12:43:12.674(1)? ufs: store created at 
I20161226-12:43:12.674(1)? ufs: store created at 
W20161226-12:43:12.679(1)? (STDERR) ufs: permissions are not defined for store "fileSystem"
W20161226-12:43:12.679(1)? (STDERR) ufs: permissions are not defined for store "fileSystem"
W20161226-12:43:12.679(1)? (STDERR) { [Error: Cannot find module './build/Release/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
W20161226-12:43:12.680(1)? (STDERR) { [Error: Cannot find module './build/default/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
W20161226-12:43:12.680(1)? (STDERR) { [Error: Cannot find module './build/Debug/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
I20161226-12:43:14.283(1) (rocketchat_migrations.js:159) Migrations: Not migrating, already at version 76
I20161226-12:43:15.684(1)? Updating process.env.MAIL_URL
I20161226-12:43:15.758(1)? Using GridFS for custom emoji storage
I20161226-12:43:15.918(1)? [Mon Dec 26 2016 12:43:15 GMT+0100 (CET)] WARNING A script has tried registering a HTTP route while the HTTP server is disabled with --disabled-httpd.
I20161226-12:43:15.921(1)? Loaded hubot-help/src/help.coffee
I20161226-12:43:15.938(1)? Loaded hello.coffee
I20161226-12:43:15.951(1)? Loaded zen.coffee
I20161226-12:43:15.960(1)? rocketchat_settings Falling back observe to model with query: { _id: 
I20161226-12:43:15.961(1)?    { '$in': 
I20161226-12:43:15.961(1)?       [ 'InternalHubot_Username',
I20161226-12:43:15.962(1)?         'InternalHubot_Enabled',
I20161226-12:43:15.962(1)?         'InternalHubot_ScriptsToLoad' ] } }
W20161226-12:43:16.622(1)? (STDERR) ufs: permissions are not defined for store "rocketchat_uploads"
I20161226-12:43:16.629(1)? Using GridFS for Avatar storage
```

--- a/packages/rocketchat-ui-admin/admin/admin.coffee
+++ b/packages/rocketchat-ui-admin/admin/admin.coffee
@@ -314,14 +314,6 @@ Template.admin.events
 			group: group
 			changed: true
 
-		if @section is ''
-			query.$or = [
-				{section: ''}
-				{section: {$exists: false}}
-			]
-		else
-			query.section = @section
-
 		settings = TempSettings.find(query, {fields: {_id: 1, value: 1, editor: 1}}).fetch()
 
 		if not _.isEmpty settings

